src/Tickets/Commerce/Admin/Singular_Order_Notices.php

```diff
@@ -0,0 +1,240 @@
+<?php
+/**
+ * Singular order status notices.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+
+namespace TEC\Tickets\Commerce\Admin;
+
+use Tribe__Admin__Notices;
+
+/**
+ * Class Singular_Order_Notices
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+class Singular_Order_Notices {
```

Comment:
I would love to chat tomorrow about this particular implementation before we merge, I have some thoughts on potentially using some other pattern that doesn't involve as many constants.